components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -445,18 +483,25 @@ public class LegacyJobOrchestrator : IJobOrchestrator, IPublisherConfigServices,
 
                 // fire config update so that the worker supervisor pickes up the changes ASAP
                 TriggerAgentConfigUpdate();
+                statusResponse.Add($"{nameof(PublishNodesAsync)}# Publishing successful for EndpointUrl: {request.EndpointUrl}");
             }
-            catch (MethodCallStatusException) {
-                throw;
+            catch (MethodCallStatusException e) {
+                statusMessage = $"{nameof(PublishNodesAsync)}# EndpointUrl: {request.EndpointUrl} Exception: {e.ResponsePayload}";
```

Comment:
The exception handling is already taken care of in the module router and works just fine. So simply rethrow the MethodCallStatusException here.